examples/client_microsoft_auth/client_msal_auth.js

```diff
@@ -0,0 +1,32 @@
+'use strict'
+
+const mc = require('minecraft-protocol')
+
+if (process.argv.length < 4 || process.argv.length > 6) {
+  console.log('Usage : node echo.js <host> <port> <email>')
+  process.exit(1)
+}
+
+const client = mc.createClient({
+  host: process.argv[2],
+  port: parseInt(process.argv[3]),
+  username: process.argv[4], // your microsoft account email
+  //   password: process.argv[5], // your microsoft account password
```

Comment:
Maybe remove this because with msal you dont specify an account password.

Although it would just get ignored either way.

Reply:
Yes, there's a lot of debug code at the moment, just leaving it there for now as the PR is still a WIP

Reply:
fixed